### PR TITLE
Add mw pin to the map

### PR DESCRIPTION
**Pull request by maartenwalter**
adding me to map

### CODE REVIEW

**Comment by leoxnidas** [comment]
Hello, you forgot to let a space between a semicolon and the number at the line 3.

File: _pins/maartenwalter.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: maartenwalter
+latitude:44.949747
```
Comment (crichID) [inline _pins/maartenwalter.json]: @maartenwalter just add a space between the `:` and the coordinate and your tests should pass! If you need any help, check out this video: https://vimeo.com/201011288

**Comment by karthikeya1992** [comment]
Hi

**Comment by brianamarie** [comment]
@maartenwalter  I made some changes so the tests will pass. Once everything turns green, I'll go ahead and merge this in since it's been a few days and we want to keep the repository clean. 😄 ✨

**Pull request merged**